packages/common/src/api/tan-query/wallets/useTokenBalance.ts

```diff
@@ -47,22 +46,18 @@ export const getTokenBalanceQueryKey = (
  */
 export const useTokenBalance = ({
   token,
```

Comment:
just checking we dont need this method for arbitrary users?

Reply:
So we dropped the incoming userid so just confirming we don't need it for any user?